examples/README.md

```diff
@@ -9,8 +9,11 @@ You may want to read through the
 [Quick Start Guide](https://grpc.io/docs/quickstart/java.html)
 before trying out the examples.
 
-To build the examples, run in this directory:
+To build the examples,
 
+1. **(Only need this step for non-released versions, e.g. master HEAD) [Install gRPC Java 1.18.0-SNAPSHOT library locally](../COMPILING.dm).** <!--- CURRENT_GRPC_VERSION -->
```

Comment:
2 points:

- can we avoid "hardcoding" 1.18.0-SNAPSHOT in this README file? People may not remember changing this when the HEAD points to something newer.

- put the text in parentheses after the main sentence i.e.

 [Install gRPC Java 1.18.0-SNAPSHOT library locally](../COMPILING.dm) (Only need this step for non-released versions, e.g. master HEAD)